src/FiberTraces.cc

```diff
@@ -2716,16 +2716,18 @@ namespace pfsDRPStella = pfs::drp::stella;
       }
 
       int y = ftf->yCenter + ftf->yLow;
-      for (int iY = 0; iY < xCenters.getShape()[ 0 ]; ++iY){
-        for (int x = minCenMax[ iY ][ 0 ]; x <= minCenMax[ iY ][ 2 ]; ++x){
-          ccdArray[ y + iY ][ x ] += fiberTraceRepresentation.getArray()[ iY ][ x - minCenMax[ iY ][ 0 ] ];
-        }
-      }
-      return true;
+      ndarray::Array<size_t, 2, 1> xMinMax = ndarray::allocate(minCenMax.getShape()[0], 2);
+      xMinMax[ndarray::view()(0)] = minCenMax[ndarray::view()(0)];
+      xMinMax[ndarray::view()(1)] = minCenMax[ndarray::view()(2)];
+      addArrayIntoArray(fiberTraceRepresentation.getArray(),
+                        xMinMax,
+                        y,
+                        ccdArray);
+      return;
```

Comment:
Point taken, but I leave it as it is for now. If you think it should get changed I can file a ticket...?